index.markdown

```diff
@@ -1,10 +1,75 @@
 ---
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
 layout: default
 ---
 
-# Big Test Energy
+# UWB Hacks 2020 Documentation
+
+![logo](https://cdn.discordapp.com/attachments/652802883531571210/668393411564404736/websiteDesign1.png)
+Temporary logo placeholder using an awesome design created by Erica
+
+Welcome to the documentation page for UWB ACM's annual hackathon! Check out 
+some of the guides below to prepare for the big day, find inspiration
+for potential projects, or get help if you are feeling lost or stuck. 
+
+
+## Schedule
+  
+TODO: This is mostly pulled from last year's schedule
```

Comment:
Double-check this schedule and update it when we finalize the schedule for this year's hackathon